msteams-platform/bots/how-to/authentication/add-authentication.md

```diff
@@ -629,6 +629,15 @@ Within the following dialog step, check for the presence of a token in the resul
 
 ---
 
+## Code sample
+
+This section provides Bot authentication v3 SDK sample.
+
+| **Sample name** | **Description** | **.NET** | **Node.js** | **Python** |
+|---------------|------------|------------|-------------|---------------|
+| Bot authentication | This sample shows how to get started with authentication in a bot for Microsoft Teams. | [View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/46.teams-auth) | [View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/46.teams-auth) | [View](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/python/46.teams-auth) |
```

Comment:
On line 634 we say these are v3 sample, but this sample is based on Bot Framework v4